### PR TITLE
Update SurveyModel doccomments, part 1

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

File: src/survey.ts [hunk 1]
```diff
@@ -75,10 +75,15 @@ export class SurveyModel extends SurveyElementCore
     return SurveyModel.platform;
   }
   /**
-   * You can display an additional field (comment field) for the most of questions; users can enter additional comments to their response.
-   * The comment field input is saved as `'question name' + 'commentSuffix'`.
-   * @see data
-   * @see Question.showCommentArea
+   * A suffix added to the name of the property that stores comments.
+   *
+   * Default value: "Comment"
```
Comment (andrewtelnov) [inline src/survey.ts:80]: Default value: "-Comment"

File: src/survey.ts [hunk 2]
```diff
@@ -105,179 +110,271 @@ export class SurveyModel extends SurveyElementCore
...
    */
   public onShowingPreview: EventBase<SurveyModel> = this.addEvent<
     SurveyModel
   >();
   /**
-   * The event is fired after a user clicks the 'Complete' button. The event allows you to specify the URL opened after completing a survey.
-   * Specify the `navigateToUrl` property to make survey navigate to another url.
-   *- `sender` - the survey object that fires the event.
-   *- `options.url` - Specifies a URL opened after completing a survey. Set this property to an empty string to cancel the navigation and show the completed survey page.
+   * An event that is raised before the survey navigates to a specified URL. Use this event change the URL or cancel the navigation.
```
Comment (annnke) [inline src/survey.ts:181]:
```suggestion
   * An event that is raised before the survey navigates to a specified URL. Use this event to change the URL or cancel the navigation.
```

File: src/survey.ts [hunk 3]
```diff
@@ -105,179 +110,271 @@ export class SurveyModel extends SurveyElementCore
...
-   *- `option.isPrevPage` - commonly means, that end-user press the previous page button. In general, it means that options.newCurrentPage is the previous page before options.oldCurrentPage
-   * @see currentPage
+   * An event that is raised before the current page is switched.
+   *
+   * Parameters:
+   *
+   * - `sender`: `SurveyModel`\
+   * A survey instance that raised the event.
+   * - `options.oldCurrentPage`: [`PageModel`](https://surveyjs.io/form-library/documentation/api-reference/page-model)\
+   * A page used to be current.
```
Comment (annnke) [inline src/survey.ts:222]:
```suggestion
   * The current page.
```

File: src/survey.ts [hunk 4]
```diff
@@ -105,179 +110,271 @@ export class SurveyModel extends SurveyElementCore
...
-   *- `option.isPrevPage` - commonly means, that end-user press the previous page button. In general, it means that options.newCurrentPage is the previous page before options.oldCurrentPage
-   * @see currentPage
+   * An event that is raised after the current page is switched.
+   *
+   * Parameters:
+   *
+   * - `sender`: `SurveyModel`\
+   * A survey instance that raised the event.
+   * - `options.oldCurrentPage`: [`PageModel`](https://surveyjs.io/form-library/documentation/api-reference/page-model)\
+   * A page used to be current.
```
Comment (annnke) [inline src/survey.ts:246]:
```suggestion
   *A page that used to be current.
```

File: src/survey.ts [hunk 5]
```diff
@@ -105,179 +110,271 @@ export class SurveyModel extends SurveyElementCore
...
+   * - `options.newCurrentPage`: [`PageModel`](https://surveyjs.io/form-library/documentation/api-reference/page-model)\
+   * A page that became current.
```
Comment (annnke) [inline src/survey.ts:248]:
```suggestion
   * The current page.
```

File: src/survey.ts [hunk 6]
```diff
@@ -105,179 +110,271 @@ export class SurveyModel extends SurveyElementCore
...
+   * An event that is raised before a question value is changed.
+   *
+   * Parameters:
+   *
+   * - `sender`: `SurveyModel`\
+   * A survey instance that raised the event.
+   * - `options.name`: `String`\
+   * The `name` of the question whose value is being changed. If you use the [`valueName`](https://surveyjs.io/form-library/documentation/api-reference/text-entry-question-model#valueName) property, this parameter contains its value.
+   * - `options.question`: [`Question`](https://surveyjs.io/form-library/documentation/api-reference/question)\
+   * The question whose value is being changed. If you use `valueName`, and it is the same for several questions, this parameter contains the first question.
```
Comment (annnke) [inline src/survey.ts:270]:
```suggestion
   * The question whose value is being changed. If you use `valueName` and it is the same for several questions, this parameter contains the first question.
```

File: src/survey.ts [hunk 7]
```diff
@@ -105,179 +110,271 @@ export class SurveyModel extends SurveyElementCore
...
-   * or inside a matrix cell or it can be a text question in multiple text items or inside a panel of a panel dynamic.
-   * You can use this event to set up properties to a question based on it's type for all questions, regardless where they are located, on the page or inside a matrix cell.
-   * Please note: If you want to use this event for questions loaded from JSON then you have to create survey with empty/null JSON parameter, assign the event and call survey.fromJSON(yourJSON) function.
+   * An event that is raised when the survey creates any new object derived from [`Question`](https://surveyjs.io/form-library/documentation/api-reference/question).
+   *
+   * In a survey, complex elements ([Dynamic Matrix](https://surveyjs.io/form-library/examples/questiontype-matrixdynamic/), [Multiple Text](https://surveyjs.io/form-library/examples/questiontype-multipletext/), [Dynamic Panel](https://surveyjs.io/form-library/examples/questiontype-paneldynamic/)) are composed from questions. Use this event to customize any question regardless of which survey element it belongs to.
```
Comment (annnke) [inline src/survey.ts:367]:
```suggestion
   * In a survey, complex elements ([Dynamic Matrix](https://surveyjs.io/form-library/examples/questiontype-matrixdynamic/), [Multiple Text](https://surveyjs.io/form-library/examples/questiontype-multipletext/), and [Dynamic Panel](https://surveyjs.io/form-library/examples/questiontype-paneldynamic/)) are composed of questions. Use this event to customize any question regardless of which survey element it belongs to.
```

File: src/survey.ts [hunk 8]
```diff
@@ -423,16 +589,18 @@ export class SurveyModel extends SurveyElementCore
   >();
   /**
    * Use this event to modify the HTML content before rendering, for example `completeHtml` or `loadingHtml`.
-   *- `options.html` - specifies the modified HTML content.
+   * - `sender`: `SurveyModel` - A survey instance that raised the event.
+   * - `options.html` - an HTML that you may change before text processing and then rendering. specifies the modified HTML content.
```
Comment (annnke) [inline src/survey.ts:593]:
```suggestion
   * - `options.html` - HTML that you can change before text processing and rendering. Specifies modified HTML content.
```

File: src/survey.ts [hunk 9]
```diff
@@ -552,158 +720,162 @@ export class SurveyModel extends SurveyElementCore
...
-   * - `options.item` - The choice item as specified in the [choices](https://surveyjs.io/Documentation/Library?id=QuestionSelectBase#choices) array.
-   * - `options.visible` - A Boolean value that specifies the item visibility. Set it to `false` to hide the item.
+   * - `sender`: `SurveyModel`\
+   * A survey instance that raised the event.
+   * - `options.question`\
+   * A Question instance to which the choice item belongs.
+   * - `options.item`\
+   * The choice item as specified in the [choices](https://surveyjs.io/Documentation/Library?id=QuestionSelectBase#choices) array.
+   * - `options.visible`\
+   * A Boolean value that specifies the item visibility. Set it to `false` to hide the item.
```
Comment (annnke) [inline src/survey.ts:878]:
```suggestion
   * A Boolean value that specifies item visibility. Set it to `false` to hide the item.
```

File: src/survey.ts [hunk 10]
```diff
@@ -914,78 +1094,98 @@ export class SurveyModel extends SurveyElementCore
...
+   * An event that is raised after an item value is changed in a panel within a [Dynamic Panel](https://surveyjs.io/form-library/examples/questiontype-paneldynamic/) question.
+   *
+   * Parameters:
+   *
+   * - `sender`: `SurveyModel`\
+   * A survey instance that raised the event.
+   * - `options.question`: [`QuestionPanelDynamicModel`](https://surveyjs.io/form-library/documentation/api-reference/dynamic-panel-model)\
+   * A Dynamic Panel question.
+   * - `options.panel`: [`PanelModel`](https://surveyjs.io/form-library/documentation/api-reference/panel-model)\
+   * A panel that nests the item with changed value.
```
Comment (annnke) [inline src/survey.ts:1170]:
```suggestion
   * A panel that nests the item with a changed value.
```

File: src/survey.ts [hunk 11]
```diff
@@ -1582,8 +1783,10 @@ export class SurveyModel extends SurveyElementCore
     this.setPropertyValue("navigateToUrl", val);
   }
   /**
-   * Gets or sets a list of URL condition items. If the expression of this item returns `true`, then survey will navigate to the item URL.
-   * @see UrlConditionItem
+   * An array of objects that allows you to navigate respondents to different URLs after survey completion.
+   *
+   * Each object should include the [`expression`](https://surveyjs.io/form-library/documentation/api-reference/urlconditionitem#url) and [`url`](https://surveyjs.io/form-library/documentation/api-reference/urlconditionitem#expression) properties. When an `expression` evaluates to `true`, the survey navigates to the corresponding `url`. Refer to the following help topic for more information about expressions: [Expressions](https://surveyjs.io/form-library/documentation/design-survey/conditional-logic#expressions).
```
Comment (annnke) [inline src/survey.ts:1788]:
```suggestion
   * Each object should include the [`expression`](https://surveyjs.io/form-library/documentation/api-reference/urlconditionitem#url) and [`url`](https://surveyjs.io/form-library/documentation/api-reference/urlconditionitem#expression) properties. When `expression` evaluates to `true`, the survey navigates to the corresponding `url`. Refer to the following help topic for more information about expressions: [Expressions](https://surveyjs.io/form-library/documentation/design-survey/conditional-logic#expressions).
```

**Review by annnke — APPROVED** [review]

**Pull request merged**